libsolidity/interface/ReadFile.h

```diff
@@ -37,8 +37,21 @@ class ReadCallback: boost::noncopyable
 		std::string responseOrErrorMessage;
 	};
 
+	enum class Kind
+	{
+		ReadFile,
+		SMTQuery
+	};
+
+	static std::string kindString(Kind _kind)
+	{
+		if (_kind == Kind::SMTQuery)
+			return "smt-query";
+		return "source";
```

Comment:
Why not a switch statement which would catch unmatched enum cases?

Reply:
Fixed